src/utils/stackingUtils.ts

```diff
@@ -20,7 +24,10 @@ export type IStackingOrder = "topdown" | "bottomup";
 
 // HACKHACK More accurate stacking result definition. Remove typeless stacking result
 // and replace with typed stacking result in 3.0.
-export type GenericStackingResult<D> = Utils.Map<Dataset, Utils.Map<D, StackedDatum>>;
+//
+// Maps compare Date keys using referential equality. So, for safety, we only want primitives
```

Comment:
Ugh that's annoying. The handling of Date vs number vs string in Plottable seems wonky and all over the place. Are we re-creating different `new Date()` objects for the same underlying data?